src/simple_openid_connect/integrations/django/views.py

```diff
@@ -50,11 +39,11 @@ class InitLoginView(View):
     """
 
     def get(self, request: HttpRequest) -> HttpResponse:
+        if "next" in request.GET.keys():
+            request.session["next"] = request.GET["next"]
         logout(request)
         client = OpenidAppConfig.get_instance().get_client(request)
-        redirect = client.authorization_code_flow.start_authentication(
-            additional_redirect_args=_get_redirect_args(request),
```

Comment:
As far as I remember, this was the only reason I added `additional_redirect_args` in the first place. Since we now know it to not be of much use, I would like to remove it from `start_authentication()` to keep the API simple.